### ISSUE TITLE
Finetune Transformers Models with PyTorch Lightning: documentation error?

### ISSUE THREAD

**Issue by yfeng24816** [PyTorchLightning/lightning-tutorials#139]

**Comment by yfeng24816** [comment]
> 

So do you also think there is an error in the documentation too? I am not sure by my own.

**Comment by Borda** [comment]
@stancld mind have a look, please? :chipmunk:

**Comment by stancld** [comment]
@Borda Yes, I will have a look on Friday (tmrw). Can you assign the issue to me, please? :]

**Comment by stancld** [comment]
It looks like no such example is present in tutorial anymore 🤔

**Comment by Borda** [comment]
> It looks like no such example is present in tutorial anymore 🤔

https://github.com/Lightning-AI/tutorials/tree/main/lightning_examples/text-transformers

**Comment by stancld** [comment]
Oh sorry, I have forked `tutorials` repo, but it's `pytorch` one :D

**Comment by stancld** [comment]
@Borda I checked the notebooks and it looks like the calculation of the total number of training steps is now the responsibility of a Lightning Trainer. The reported error is, therefore, no more relevant for this example, and I believe the issue can be closed/marked as done.
